client/src/components/Activities.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {getActivities} from '../actions/activityActions';
+import {connect} from 'react-redux';
+import Slider from 'react-slick';
+import Comments from "./Comments";
+// import "slick-carousel/slick/slick.css";
+// import "slick-carousel/slick/slick-theme.css";
+
+class Activities extends Component {
+    constructor() {
+      super();
+      this.state = {
+        Activities: []
+      };
+    }
+
+
+componentDidMount() {
```

Comment:
It looks like the spacing is generally off in some places across the project.  I'd recommend running a tidying plug-in for spacing to clean it up and have all the files updated in a single commit.